ansible/dimensiondata/dimensiondata_backup.py

```diff
@@ -38,8 +38,11 @@
   region:
     description:
       - The target region.
-    choices: %s
-    default: na
+    choices:
+      - Regions are defined in Apache libcloud project file = libcloud/common/dimensiondata.py 
+      - See https://libcloud.readthedocs.io/en/latest/compute/drivers/dimensiondata.html
+      - Note: Values avail in array dd_regions(). 
+    default: na aka North America
```

Comment:
Using:      default: 'na' # (North America), the comment doesn’t show up in the documentation

If I remove the #, then it fails to generate documentations.
So I think what I have is a close approx of what we want.